ocsp/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "ocsp"
```

Comment:
The `ocsp` crate name is taken, however you can use this:
```suggestion
name = "x509-ocsp"
```

Reply:
Changed.